spec/models/links_udpate_spec.rb

```diff
@@ -53,28 +53,43 @@
   end
 
   describe '#mark_as_errored' do
-    let(:tag_mapping) { create(:tag_mapping) }
+    let!(:tag_mapping) { create(:tag_mapping) }
 
     before do
-      links_update.tag_mappings = TagMapping.all
+      publishing_api_has_lookups('/my-base-path' => 'my-content-id')
+
+      publishing_api_has_linkables(
+        [
+          { 'title' => 'Early Years', content_id: 'my-content-id' },
+        ],
+        document_type: 'taxon'
+      )
     end
 
-    it 'marks a number of tag mappings as errored' do
-      expectation = lambda do
-        links_update.mark_as_errored
-        tag_mapping.reload
-      end
+    it 'adds error message for invalid link type' do
+      links_update = build(:links_update, links: { 'invalid' => ['my-content-id'] })
+      links_update.tag_mappings = TagMapping.all
 
-      expect { expectation.call }.to change { tag_mapping.state }.to('errored')
+      links_update.valid?
+      links_update.mark_as_errored
+      tag_mapping.reload
+
+      expect(tag_mapping.state).to eql('errored')
+      expect(tag_mapping.message).to_not match(/^Link types/)
```

Comment:
Let's remove this assertion that the message is no longer in the previous style. We've changed the behaviour. Asserting what it _should_ be is sufficient.